### PR TITLE
refactor and document sheetmapper

### PR DESCRIPTION
**Pull request by chriswhong**
Adds Sheetmapper, an HTML template for quickly creating a Mapbox GL JS marker map with popups using live data from a Google Sheet.

Sheetmapper is an older project and was formerly part of our [impact tools](https://github.com/mapbox/impact-tools) collection. This work refactors it with the following updates:
- Uses markers instead of a `circle` layer to symbolize the features. Simple customization is possible with `color` and `scale` marker options.
- Updates the code to use async/await patterns for easier readability, less callback clutter.
- Refactors the initialization, loading the data before instantiating the map.  This allows for loading the map with the full bounds of the dataset in view, eliminating a clumsy flyto from a hard-coded initial camera view.
- Consolidates much of the configuration into a tidy configuration object for easier instructions.
- Adds a detailed README aimed at beginners who may not have much web development experience.
- Adds an easily configurable title and description overlay. Includes a media query to make the overlay full width on smaller browsers.

127_0_0_1_5500_index_html

### CODE REVIEW

File: projects/sheetmapper/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+#### 4: Run the HTML File in Your Browser
+1. With the HTML file open, right-click in the editor window and select **Open with Live Server**.
+2. Your default browser will open, displaying the HTML page. Any changes you make and save will automatically refresh in the browser. You will see a browser alert with an error if you have not yet configured Sheetmapper.
+
+## Troubleshooting Tips
+- Ensure that **Live Server** is properly installed and enabled if using the live preview.
+- If the file does not open in your browser, check the file path or try opening it manually by dragging it into your browser window.
+
+### Step 4. Configure Sheetmapper
```
Comment (AndrewSepic) [inline projects/sheetmapper/README.md:87]: I think this step should be moved up to above Step 3 - INstall Live Server.  I don't see why we should run a file in the browser before it has been configured.

**Review by AndrewSepic — APPROVED** [review]
Approving with 2 comments!

File: projects/sheetmapper/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+2. Click on the **Raw** button to view the raw HTML code.
+3. Right-click on the page and select **Save as…** to save the HTML file to your computer.
+
+#### 2: Open the HTML File in Visual Studio Code
+1. Open **Visual Studio Code** on your computer.
+2. Use **File > Open File…** or **File > Open Folder…** to open the Sheetmapper HTML file or the folder containing the Sheetmapper HTML file.
+3. Once the file is open, you should see the HTML code in Visual Studio Code.
+
+#### 3: Install the Live Server Extension
+To make it easier to preview changes in real-time, you can install the Live Server extension in Visual Studio Code:
```
[the remark this answers is not in the document]
Comment (chriswhong) [reply projects/sheetmapper/README.md:74]: Will do!

File: projects/sheetmapper/README.md (hunk 1 above)
Comment (chriswhong) [reply projects/sheetmapper/README.md:87]: Will do!

**Pull request merged**